include/bdldl/bdldl.h

```diff
@@ -13,13 +13,27 @@
 
 #include <stddef.h>
 
-#include "sparseLDL/sparseLDL_forward_declaration.h"
-#include "sparseLDL/sparseLDL_types.h"
+#include "bdldl/bdldl_types.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/////////////////////////////////////////////
+// Forward declaration
+/////////////////////////////////////////////
+
+struct ldl_Solver;
+
+/////////////////////////////////////////////
+// Solver status
+/////////////////////////////////////////////
+
+typedef enum {
+  OK,
```

Comment:
Since C enums aren't prefixed by the enum name, we should explicitly prefix these enum values, e.g. `bdldl_OK`, 'bdldl_ERROR'.

We might also want to add a few other statuses in here that we know we're likely to encounter, like
* out of memory error
* out-of-bounds error
* non-singular matrix
* dimension mismatch
Since you're more familiar with the algorithm than I am, I'll let you decide what makes sense to add in here.

nit: explicitly specify `OK = 0` for clarity and to make sure it never changes on accident (by adding something before `OK`.